tests/plots/test_patheffects.py

```diff
@@ -23,6 +26,49 @@ def test_fronts():
     return fig
 
 
+@pytest.mark.mpl_image_compare(remove_text=True, tolerance=0.)
+def test_frontogenesis():
+    """Test plotting forming front path effects."""
+    x = np.linspace(0, 80, 5)
+    fig, ax = plt.subplots(figsize=(12, 6), dpi=300)
+    ax.plot(x, np.ones_like(x), path_effects=[ColdFrontogenesis(spacing=1.0)])
+    ax.plot(x, np.ones_like(x) * 3, path_effects=[WarmFrontogenesis(flip=True, spacing=1.0)])
+    ax.plot(x, np.ones_like(x) * 5, path_effects=[OccludedFrontogenesis(spacing=1.0,
+                                                                        color='black')])
+    ax.plot(x, np.ones_like(x) * 7, path_effects=[StationaryFrontogenesis(spacing=1.0)])
+    ax.set_ylim(-0.5, 8)
+    return fig
+
+
+@pytest.mark.mpl_image_compare(remove_text=True, tolerance=0.)
+def test_frontolysis():
+    """Test plotting dissipating front path effects."""
+    x = np.linspace(0, 80, 5)
+    fig, ax = plt.subplots(figsize=(12, 6), dpi=300)
+    ax.plot(x, np.ones_like(x), path_effects=[ColdFrontolysis(spacing=1.0)])
+    ax.plot(x, np.ones_like(x) * 3, path_effects=[WarmFrontolysis(spacing=1.0,
+                                                                  color='orange')])
+    ax.plot(x, np.ones_like(x) * 5, path_effects=[OccludedFrontolysis(spacing=1.0)])
+    ax.plot(x, np.ones_like(x) * 7, path_effects=[StationaryFrontolysis(flip=True,
+                                                                        spacing=1.0)])
+    ax.set_ylim(-0.5, 8)
+    return fig
+
+
+@pytest.mark.mpl_image_compare(remove_text=True, tolerance=0.)
```

Comment:
To pass on older matplotlib for some reason (it only dislikes `Squall`):
```suggestion
@pytest.mark.mpl_image_compare(remove_text=True, tolerance=0.023)
```